src/main/java/emu/grasscutter/command/commands/TeleportCommand.java

```diff
@@ -1,67 +1,88 @@
 package emu.grasscutter.command.commands;
 
+import emu.grasscutter.Grasscutter;
 import emu.grasscutter.command.Command;
 import emu.grasscutter.command.CommandHandler;
 import emu.grasscutter.game.player.Player;
 import emu.grasscutter.utils.Position;
 
 import java.util.List;
 
-@Command(label = "teleport", usage = "teleport <x> <y> <z>", aliases = {"tp"},
+@Command(label = "teleport", usage = "teleport [@player id] <x> <y> <z> [scene id]", aliases = {"tp"},
         description = "Change the player's position.", permission = "player.teleport")
 public final class TeleportCommand implements CommandHandler {
 
     @Override
     public void execute(Player sender, List<String> args) {
-        if (sender == null) {
-            CommandHandler.sendMessage(null, "Run this command in-game.");
+        int target;
+        if (args.size() < (sender == null ? 4 : 3)) {
+            CommandHandler.sendMessage(sender, sender == null ? "Usage: /tp @<player id> <x> <y> <z> [scene id]" :
+                    "Usage: /tp [@<player id>] <x> <y> <z> [scene id]");
             return;
         }
+        if (args.get(0).startsWith("@")) {
```

Comment:
Stay consistent with other commands and dont use @, but just the UID

Reply:
How do I handle scene id and uid if I don't use the @ prefix?

Reply:
The order of the parameters: I recommend letting xyz go first, then scene id and then UID. You can check getPlayerByUID

Reply:
But scene id should be optional, just like UID.